applications/portal/frontend/src/components/CustomAccordionSummary.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+    Box,
+    AccordionSummary,
+    Switch,
+    FormControlLabel,
+    Popover
+} from '@material-ui/core';
+
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import UP_ICON from "../assets/images/icons/up.svg";
+import { POPULATION_FINISHED_STATE } from "../utilities/constants";
+import { getRGBAFromHexAlpha, getRGBAString } from "../utilities/functions";
+import ColorPicker from "./common/ColorPicker";
+import SwitchLabel from "./common/SwitchLabel";
+import { TrailIcon, TrailEndIcon } from './icons';
+
+
+const POPULATION_ICONS_OPACITY = 0.4
+
+const CustomAccordionSummary = ({
+    id,
+    data,
+    pId,
+    isExpanded,
+    populations,
```

Comment:
@Aiga115 populations isn't used activily, just populations[pid] 
suggestion: only pass the population as a param